src/base/bittorrent/session.h

```diff
@@ -729,11 +731,12 @@ namespace BitTorrent
 
         int m_numResumeData = 0;
         int m_extraLimit = 0;
+        int m_globalDownloadRate = 0;
+        int m_globalUploadRate = 0;
```

Comment:
I'm not an authority on this code base, but it seems like the `global` prefix is misleading here. Are there other transfer rates stored in the session? If anything, I would call these `payloadDownloadRate` and `payloadUploadRate`, since these counters only count the payload, not protocol overhead.